networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -3,6 +3,59 @@
 input AMPLIFY {
   globalAuthRule: AuthRule = { allow: public }
 } # FOR TESTING ONLY!
+
+type GameTemplate @model{
+  id: String!
+  title: String!
+  owner: String! @index(
+      name: "gameTemplatesByOwner"
+      queryField: "gameTemplatesByOwner"
+      sortKeyFields: ["createdAt"]
+    )
+  version: Int!
+  description: String!
+  domain: String
+  cluster: String
+  grade: String
+  standard: String
```

Comment:
These are indeed based off the questions. When we first load the page in `central` this information needs to be displayed on a game-by-game basis. Because we have a connection between the gameTemplates and questionTemplates, we could derive the game-level CCSS from this information but we'd need to do it every time the page is loaded and then paginated. Is that more efficient than storing it in the db anyways though?